configure

```diff
@@ -202,7 +202,7 @@ function install_pkgin
 	local pkglist
 
 	pkglist="build-essential flex libxslt openjdk11 nodejs"
-	pkglist="$pkglist p5-XML-Parser gettext python27 py27-expat"
+	pkglist="$pkglist p5-XML-Parser gettext python27 py27-expat python39"
```

Comment:
It might be worth simplifying this to just pull in `py27-expat` and `py27-sqlite3`, they will pull in `python27` automatically as a dependency so no need to list it explicitly.  Also are they still required now?

Obviously these are unrelated to your change so feel free to ignore if this should go into a separate ticket, just thought I'd mentioned it in passing.

Reply:
Ooof, I hadn't thought about that.

And yes py27-sqlite3 is required, see OS-6660 for details.